test/lib/jsonStathamSpec.js

```diff
@@ -2,120 +2,112 @@ import utils from '../../test/support/TestUtils'
 import jsonStatham from'../../src/lib/jsonStatham'
 import AjaxAdaptorBase from '../../src/lib/AjaxAdaptorBase'
 
-describe('jsonStatham', function() {
-  var listener
+
+var spy
+const args = () => spy.getCall(0).args[0]
```

Comment:
small detail, but something esoteric like this is helpful to have closer to the context it is used, so it's clear what it's for. Granted it started with a bad name to begin with...  

It's also a little gnarly of a scope, since spy keeps getting re-assigned in the before each, args will return a different value based on who's calling it. Feel like it should be closer to the beforeEach block. (having it at the top makes it feel more like a static thing).